docs/reference-guides/cluster-configuration/rancher-server-configuration/k3s-cluster-configuration.md

```diff
@@ -15,141 +15,421 @@ You can configure the Kubernetes options one of two ways:
 
+## Editing Clusters in the Rancher UI
+
+The Rancher UI provides two ways to edit a cluster:
+1. With a form.
+1. With YAML.
+
+### Editing Clusters with a Form
+
+The form covers the most frequently needed options for clusters.
+
+To edit your cluster,
+
+1. Click **☰ > Cluster Management**.
+1. Go to the cluster you want to configure and click **⋮ > Edit Config**.
+
+### Editing Clusters in YAML
+
+Rancher UI provides two ways to edit cluster: editing clusters with a form or in YAML.
+For a complete reference of configurable options for K3s clusters in YAML, see the [K3s documentation.](https://rancher.com/docs/k3s/latest/en/installation/install-options/)
+
+To edit your cluster with YAML:
+
+1. Click **☰ > Cluster Management**.
+1. Go to the cluster you want to configure and click **⋮ > Edit as YAML**.
+1. Edit the RKE options under the `rkeConfig` directive.
+
 ## Configuration Options in the Rancher UI
 
-:::tip
+### Machine Pool Configuration
 
-Some advanced configuration options are not exposed in the Rancher UI forms, but they can be enabled by editing the K3s cluster configuration file in YAML. For the complete reference of configurable options for K3s clusters in YAML, see the [K3s documentation.](https://rancher.com/docs/k3s/latest/en/installation/install-options/)
+This subsection covers generic machine pool configurations. For specific infrastructure provider configurations, refer to the following:
 
-:::
+- [Azure](../downstream-cluster-configuration/machine-configuration/azure.md)
+- [DigitalOcean](../downstream-cluster-configuration/machine-configuration/digitalocean.md)
+- [EC2](../downstream-cluster-configuration/machine-configuration/amazon-ec2.md)
+
+##### Pool Name
+
+The name of the machine pool.
+
+##### Machine Count
+
+The number of machines in the pool.
+
+##### Roles
+
+Option to assign etcd, control plane, and worker roles to nodes.
+
+#### Advanced
 
-### Basics
-#### Kubernetes Version
+##### Auto Replace
+
+The duration nodes can be unreachable before they are automatically deleted and replaced.
+
+##### Drain Before Delete
+
+Enables draining nodes by evicting all pods before the node is deleted.
+
+##### Kubernetes Node Labels
+
+Add [labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) to nodes to help with organization and object selection.
+
+For details on label syntax requirements, see the [Kubernetes documentation.](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set)
+
+##### Taints
+
+Add [taints](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) to nodes, which can be used to prevent pods from being scheduled to or executed on nodes, unless the pods have matching tolerations.
```

Comment:
```suggestion
Add [taints](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) to nodes, to prevent pods from being scheduled to or executed on the nodes, unless the pods have matching tolerations.
```